src/vm/instruction_set.md

```diff
@@ -481,6 +497,22 @@ Panic if:
 
 `$err` is cleared.
 
+### MLDV: Fused multiply-divide
+
+|             |                                                                                       |
+|-------------|---------------------------------------------------------------------------------------|
+| Description | Multiplies two registers with arbitrary precision, then divides by a third register.  |
+| Operation   | `a = (b * c) / d;`                                                                    |
+| Syntax      | `mldv $rA, $rB, $rC, $rD`                                                             |
+| Encoding    | `0x00 rA rB rC rD`                                                                    |
+| Notes       | Division by zero is treated as division by `1 << 64` instead.                         |
+
+If the divisor (`$rD`) is zero, then instead the value is divided by `1 << 64`. This returns the higher half of the 128-bit multiplication result. This operation never overflows.
```

Comment:
Won't this just return `0` if `(b * c) < (1 << 64)` or `1` if `(b * c) > (1 << 64)`

Reply:
No. For instance `(u64::MAX*u64::MAX) / (1 << 64) =18446744073709551614 `